### PR TITLE
Return correct action status in case of failure

### LINKED ISSUE #4: Action should has failed status in case of problem
When something was wrong during process any repository action status should has failed status.

Should process all repository independently of status last one.

eg.
https://github.com/s4u/.github/runs/1328323546?check_suite_focus=true

```

```

### PR DESCRIPTION
**Pull request by slawekjaranowski**
fix #4

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -22,8 +22,3 @@ inputs:
 runs:
   using: 'docker'
   image: 'Dockerfile'
-  args:
```
Comment (kbrashears5) [inline action.yml:25]: why was this removed?

**Review by kbrashears5 — CHANGES_REQUESTED** [review]
didn't know that this error matcher was a thing, that's pretty cool. only issue I see is that the args have been removed in the action.yml file, get those added back and I'll get it merged

File: action.yml (hunk 1 above)
Comment (slawekjaranowski) [reply action.yml:25]: it is not used, all inputs are set as environment variable.
you don't use nowhere variable like $1, $2, ...

Comment (kbrashears5) [reply action.yml:25]: just read up on them, looks good

**Review by kbrashears5 — APPROVED** [review]
